src/Select.tsx

```diff
@@ -16,6 +16,10 @@ const StyledSelect = styled.select`
   outline: none;
   width: 100%;
 
+  option {
+    color: initial;
+  }
+
```

Comment:
Now that option is set to `color:initial`, can we delete the styles on line 28?
```
  &:invalid option:not(:first-child) {
    color: ${get('colors.fg.default')};
  }
```

Reply:
I believe so. Let me test it out...